demo/query.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+document.addEventListener('DOMContentLoaded', () => {
+  const formatObjectToString = (obj) => JSON.stringify(obj, null, 2);
+
+  const emeRunButton = document.querySelector('#emeRun');
+
+  emeRunButton.addEventListener('click', async () => {
+    // Pull contents of query form.
+    const keySystem = window.keySystem.input.value;
```

Comment:
Why use the global name and not use `document.getElementById`?  Why not use `window.emeRun` for the above?

Reply:
I'll use the global name on all of them for consistency.  But here I **must** say `window.` when accessing the elements, since the local var has the same name as the global one I'm pulling values from.